js/ui/metric-legend.js

```diff
@@ -12,14 +12,20 @@ export const MetricLegend = {
     metricPopup: function () {
       const metricBase = Store.getMetricBase(this.$props.metric.name)
       Store.setMetricPopup(metricBase, !this.$props.metric.popup)
+    },
+    trashcanClicked: function () {
+      const metricBase = Store.getMetricBase(this.$props.metric.name)
+      Store.deleteMetric(metricBase)
```

Comment:
I think this line should not be necessary. I think the other `deleteMetric` function does this implicitly.
```suggestion
```